openshift/system/contrib/scl_enable

```diff
@@ -1,4 +1,4 @@
 # This file contains automatic SCL enablement.
 unset BASH_ENV PROMPT_COMMAND ENV
-source scl_source enable rh-ruby24
+source scl_source enable rh-ruby25
```

Comment:
Has this been tested? Like `ruby -v` returning version

Reply:
yeah!

bash-4.2$ ruby -v
ruby 2.5.5p157 (2019-03-15 revision 67260) [x86_64-linux]